.yardopts

```diff
@@ -3,6 +3,8 @@
 --protected
 --no-private
 --template-path docs/yard/template
+--tag incompatibility:"Incompatibilities with Shoes"
```

Comment:
this is nice..

Reply:
I like it, yeah :-) We can add other little categories like this if we want to, but I'm not sure which others we would need.